.github/workflows/ci.yml

```diff
@@ -139,6 +141,12 @@ jobs:
         with:
           key: ${{ matrix.target }}-a # suffix for cache busting
 
+      - name: download nextest
+        if: matrix.kind == 'local'
+        shell: bash
+        run: |
+          curl -LsSf ${{ matrix.nextest_url }} | tar xzf - -C ${CARGO_HOME:-~/.cargo}/bin
```

Comment:
this is a bit scary :/

Reply:
Yeah not the couthest thing in the world, but not really any better options.